pkg/server/deploy/models.go

```diff
@@ -0,0 +1,15 @@
+package deploy
+
+import dpb "github.com/luizalabs/teresa/pkg/protobuf/deploy"
+
+type Rollback struct {
```

Comment:
I'm against creating a `Rollback` struct to hold those fields. A rollback operation is issued against an application and it's revision, not against a 'rollback struct variable'.

Packing the fields in this struct makes things unnecessarily opaque.